dashboard/app/controllers/api/v1/schools_controller.rb

```diff
@@ -7,4 +9,14 @@ def index
     end
     render json: serialized_schools
   end
+
+  # GET /dashboardapi/v1/schoolsearch/:criteria/:limit
+  def search
+    schools = School.where("name LIKE ?", "%#{params[:q].upcase}%").order(:name).limit(params[:limit])
+    #ChatClient.log "#{schools.to_s}"
+    serialized_schools = schools.map do |school|
+      Api::V1::SchoolSerializer.new(school).attributes
+    end
+    render json: serialized_schools
+  end
```

Comment:
I might be misunderstanding what @ashercodeorg is asking for, but I took his meaning as putting a fairly small upper limit on that number (maybe 100 or so, depending how many we'd expect to need to return), is that right Asher?